### ISSUE TITLE
WTForm's built-in error message for zh_TW

### ISSUE THREAD

**Issue by newnewcoder** [lepture/flask-wtf#331]
I want to use WTForm's built-in error message for [zh_TW](https://github.com/wtforms/wtforms/tree/master/wtforms/locale/zh_TW/LC_MESSAGES) with flask-Babel. However, flask-Babel always parse `zh_TW` to `zh_Hant_TW`, causing flask-wtf's i18n can't get the correct locale.

Is there any solution? or I made some mistake? thx!

**Comment by greyli** [comment]
Currently, if you want to use WTForms's built-in message, you have to use the latest version's Flask-WTF on GitHub. Here is a demo:
```py
app = Flask(__name__)

app.config['WTF_I18N_ENABLED'] = False

class MyBaseForm(FlaskForm):
    class Meta:
        locales = ['zh_TW']

class HelloForm(MyBaseForm):
    name = StringField('Name', validators=[DataRequired(), Length(1, 20)])
    body = TextAreaField('Message', validators=[DataRequired(), Length(1, 200)])
    submit = SubmitField()
```

**Comment by newnewcoder** [comment]
thank you!
this problem is solved by using version **0.15.dev20180602**
